### ISSUE TITLE
Do you need a port? Can i have examples setting please.

### ISSUE THREAD

**Issue by Bobdidit69** [UnamSanctam/SilentXMRMiner#529]
hey lovely miner I just dont know what to do for assembly and what process to inject to. Is it conhost.exe?  Also my friend got a miner and installer I just got a miner. Then I tried again and I for a checker and an uninstaller with my miner. Is this normal? I tried use the example settings but I dont known how to view them. Can I see what they are? And do you not need a port?

**Comment by UnamSanctam** [comment]
If you don't know what to do for "Assembly" then you can keep it disabled. The "Inject Into" is mostly cosmetic, you can leave it at the default "conhost.exe". The reason you're not getting a checker and uninstaller is because you're in the old miner repository, the new one is here: https://github.com/UnamSanctam/SilentCryptoMiner/. In that miner you can load the example settings in the "Main" tab by pressing the "Load" button, and yes you need a port for pools.

**Comment by Bobdidit69** [comment]
> If you don't know what to do for "Assembly" then you can keep it disabled. The "Inject Into" is mostly cosmetic, you can leave it at the default "conhost.exe". The reason you're not getting a checker and uninstaller is because you're in the old miner repository, the new one is here: https://github.com/UnamSanctam/SilentCryptoMiner/. In that miner you can load the example settings in the "Main" tab by pressing the "Load" button, and yes you need a port for pools.

Thanks for the fast reply yea I dont think I saw the recent updated version. Did the old version not have a place to put the port? Or am I having a brainfart ? I'm using the new version not silent xmr but silent crypto maybe not the latest one which I will try again Thanks for replying ill test the new one its pre compiled right? Can the miner run without admin permissions?

**Comment by UnamSanctam** [comment]
You put the port together with the URL (like always) `IP:PORT`, so add `:PORT` after your port URL. The releases are always pre-compiled yes. It can run without administrator permissions but if you run it as administrator then it has a lot of good features (that can only be done with administrator privileges).

**Comment by Bobdidit69** [comment]
> You put the port together with the URL (like always) `IP:PORT`, so add `:PORT` after your port URL. The releases are always pre-compiled yes. It can run without administrator permissions but if you run it as administrator then it has a lot of good features (that can only be done with administrator privileges).

Very grateful for fast replies 1 last question how does the miner run without administration permissions when it comes to sending the crypto back? How would it notify the pool to give you the money

**Comment by Bobdidit69** [comment]
> You put the port together with the URL (like always) `IP:PORT`, so add `:PORT` after your port URL. The releases are always pre-compiled yes. It can run without administrator permissions but if you run it as administrator then it has a lot of good features (that can only be done with administrator privileges).

1 last reply sorry lol so is the ssl/tls meant to be on for monero is that based on the pool? Read somewhere you said thats for ETC

**Comment by UnamSanctam** [comment]
>Very grateful for fast replies 1 last question how does the miner run without administration permissions when it comes to sending the crypto back? How would it notify the pool to give you the money

The miner only receives jobs from the pool and tries to solve them, when the miner solves them then it sends those completed jobs back to the pool and the pool verifies it, if the verification goes well then the pool will give you a bit of the blockchain block reward that your job helped get. The pool knows where to send the crypto you mine because you send your wallet address to the pool when you mine, so it never sends any crypto back to the miner itself but pays it out to the wallet address you told the pool. So there's not need for administrator permissions, just like how you don't need administrator permissions to use an internet browser.

>1 last reply sorry lol so is the ssl/tls meant to be on for monero is that based on the pool? Read somewhere you said thats for ETC

SSL/TLS means that it's connecting with a secure connection, just like how the https protocol works for websites, pools usually have a special pool port for SSL/TLS connections and another port for normal non-SSL stratum connection. So if you want to use SSL/TLS (good since the connection is secure) then you only need to use the special port the pool has.

**Comment by Bobdidit69** [comment]
> > Very grateful for fast replies 1 last question how does the miner run without administration permissions when it comes to sending the crypto back? How would it notify the pool to give you the money
> 
> The miner only receives jobs from the pool and tries to solve them, when the miner solves them then it sends those completed jobs back to the pool and the pool verifies it, if the verification goes well then the pool will give you a bit of the blockchain block reward that your job helped get. The pool knows where to send the crypto you mine because you send your wallet address to the pool when you mine, so it never sends any crypto back to the miner itself but pays it out to the wallet address you told the pool. So there's not need for administrator permissions, just like how you don't need administrator permissions to use an internet browser.
> 
> > 1 last reply sorry lol so is the ssl/tls meant to be on for monero is that based on the pool? Read somewhere you said thats for ETC
> 
> SSL/TLS means that it's connecting with a secure connection, just like how the https protocol works for websites, pools usually have a special pool port for SSL/TLS connections and another port for normal non-SSL stratum connection. So if you want to use SSL/TLS (good since the connection is secure) then you only need to use the special port the pool has.

Thanks so its OK if I just paste my address into the example one I saw you say something about a worker name after it. I'm using the monero one thanks

**Comment by UnamSanctam** [comment]
Yes you can just replace the wallet address, by default it has a worker name after the wallet address in the example settings. It's formatted like so: `WalletAddress.WorkerName` either you can keep the worker name (`{COMPUTERNAME}`) or you can remove it and just enter your wallet address and nothing else.

**Comment by Bobdidit69** [comment]
> Yes you can just replace the wallet address, by default it has a worker name after the wallet address in the example settings. It's formatted like so: `WalletAddress.WorkerName` either you can keep the worker name (`{COMPUTERNAME}`) or you can remove it and just enter your wallet address and nothing else.

I appreciate the fast responses the example miner I heard hashvault has a limit does this one? Thanks again

**Comment by UnamSanctam** [comment]
You mean the example pool? The example pool is 2miners which I haven't seen to have any limit, I've seen those with over 10 000 worker names. The HashVault pool does have a 100 miner limit (without using an xmrig-proxy).

**Comment by Bobdidit69** [comment]
> You mean the example pool? The example pool is 2miners which I haven't seen to have any limit, I've seen those with over 10 000 worker names. The HashVault pool does have a 100 miner limit (without using an xmrig-proxy).

Thanks a lot mate my miner is working fine but my mate says his one gives him an error saying no space allowed with watchdog or something concerning watchdog and spaces. Thanks a lot its fine I can do it for him just wondering why that hqppened

**Comment by UnamSanctam** [comment]
Yes the resource compiler used (windres) doesn't allow spaces in the names of its build paths (it has been a bug for like over 10 years), so build the miner somewhere without a space in its path. He can make a folder on the C: drive if he wants to be fully sure that he won't have any spaces, just make a folder called `build` directly on the C: drive and build the miner inside that.

**Comment by Bobdidit69** [comment]
> Yes the resource compiler used (windres) doesn't allow spaces in the names of its build paths (it has been a bug for like over 10 years), so build the miner somewhere without a space in its path. He can make a folder on the C: drive if he wants to be fully sure that he won't have any spaces, just make a folder called `build` directly on the C: drive and build the miner inside that.

OK thank you for that he fixed it lol

**Comment by Bobdidit69** [comment]
> Yes the resource compiler used (windres) doesn't allow spaces in the names of its build paths (it has been a bug for like over 10 years), so build the miner somewhere without a space in its path. He can make a folder on the C: drive if he wants to be fully sure that he won't have any spaces, just make a folder called `build` directly on the C: drive and build the miner inside that.

Hey when opening the miner on a laptop the miner doesn't show up on task manager? Even 90% cpu I don't think the miner is working. Using same setting as example except different pool

**Comment by UnamSanctam** [comment]
The example settings have "Stealth" enabled with the Task Manager inside the "Stealth Targets" so the miner will pause while Task Manager is open. So you shouldn't see any CPU usage.

**Comment by Bobdidit69** [comment]
> The example settings have "Stealth" enabled with the Task Manager inside the "Stealth Targets" so the miner will pause while Task Manager is open. So you shouldn't see any CPU usage.

Thank you thats pretty good. Does the miner stop working if logged out for example on a network of computers

**Comment by UnamSanctam** [comment]
Not if you have "Run as System" enabled since then it's running as the global System user.

**Comment by Bobdidit69** [comment]
> Not if you have "Run as System" enabled since then it's running as the global System user.

OK thanks a lot but I just saw that the pool and port might be outdated on the example settings? Not the pool but I dont see 12222 on 2minerd website I see 2222? Am I missing something or are the example ones still good?

**Comment by UnamSanctam** [comment]
2222 is the non-SSL port, 12222 is the SSL port, you can find the SSL port at the bottom of the page.
